### DISCUSSION TITLE
建议是升级到1.3.9？还是2.0.0-alpha？

### DISCUSSION THREAD

**Discussion by JunjianS** [apache/dolphinscheduler#6976] [Q&A] [answered]
目前使用的是1.3.2，有一些bug影响任务执行，想近期升级一下，建议是升级到1.3.9？还是2.0.0-alpha？

**Comment by davidzollo** [comment]
1.3.9

**Comment by davidzollo** [comment, marked as answer]
As a global project, Please update the title and desc in English, thx
